### DISCUSSION TITLE
Tutorial_MPdist had a link to a broken file, not reproducible

### DISCUSSION THREAD

**Discussion by ksetdekov** [TDAmeritrade/stumpy#835] [Help: Other Questions] [answered]
This line 
```python
fzip = ZipFile(io.BytesIO(urlopen("https://sites.google.com/site/mpdistinfo/home/words_dendrogram.zip?attredirects=0&d=1").read()))
```
in this tutorial [main/docs/Tutorial_MPdist.ipynb](https://github.com/TDAmeritrade/stumpy/blob/main/docs/Tutorial_MPdist.ipynb)
leads to a broken file. Does anybody have this file? I would be very grateful to be able to reproduce this results.

**Comment by seanlaw** [comment]
@ksetdekov Unfortunately, the link/data is not owned by us (it is placed there by the original authors). I'll try to reach out to the original authors to see if they are able to share their data but please know that there are no guarantees.

**Reply by seanlaw** [reply, marked as answer]
@ksetdekov The authors have restored the website: https://sites.google.com/site/mpdistinfo/

The links may need to be updated but can you please try again and let me know what you discover?

**Reply by ksetdekov** [reply]
I checked code, it works fine!
Also, rerun the notebook and added a fixed random seed for reproducibility + fixed connection deprecation warning in this PR: [https://github.com/TDAmeritrade/stumpy/pull/837](https://github.com/TDAmeritrade/stumpy/pull/837)
